src/webapp/components/RequestPopup.js

```diff
@@ -99,21 +83,6 @@ const RequestPopup = ({ requests, closePopup }) => {
             <HouseholdSizeChip size={meta["Household Size"]} />
 
             {meta["For Driving Clusters"] && <DrivingClusterChip />}
-
-            {meta.slackPermalink ? (
-              <DaysOpenChip
```

Comment:
i think the same thing here (tho maybe i'm missing something) - we want the days open/time sensitivity column to still show, just not need to be linked to the slack permalink

Reply:
+1. for this we just wanna remove the `Cant find slack` error message and always display the `DaysOpenChip`. days open used to rely on the slack post date, but this is no longer true (now it just relies on when the status changed in Airtable)

Reply:
Done, thanks for clarifying y'all!